examples/crm/src/types.ts

```diff
@@ -63,7 +63,7 @@ export interface Contact extends RaRecord {
 export interface ContactNote extends RaRecord {
     contact_id: Identifier;
     text: string;
-    date: string;
```

Comment:
**issue:** Not sure about that, we will have strings once we will have supabase backed